tests/test_export_sample.py

```diff
@@ -20,61 +20,99 @@ def __init__(self, name, udf={}, date_received="2024-01-01"):
 
 def test_removed_samples_skip_old_sample(mocker, capsys):
     sample = SampleMock("test_sample", date_received="2023-01-01")
-    project = ProjectMock('test_project')
+    project = ProjectMock("test_project")
 
-    mocker.patch('genologics.lims.Lims.get_projects', return_value=[project])
-    mocker.patch('genologics.lims.Lims.get_samples', return_value=[sample])
+    mocker.patch("genologics.lims.Lims.get_projects", return_value=[project])
+    mocker.patch("genologics.lims.Lims.get_samples", return_value=[sample])
 
     removed_samples(genologics.lims.Lims, sys.stdout)
     captured = capsys.readouterr()
     assert captured.out == (
-        'Datum verwijderd\tSample\tSample project\tWerklijst\tOnderzoeks nummer\t'
-        'Onderzoeks indicatie\tVerwijderd uit stap\tStatus\n'
+        "Datum verwijderd\tSample\tSample project\tWerklijst\tOnderzoeks nummer\t"
+        "Onderzoeks indicatie\tVerwijderd uit stap\tStatus\n"
     )
 
 
 def test_sample_udf_with_udf(mocker, capsys):
     # Test output for sample with known Dx-udf in database
-    sample = SampleMock("test_sample", udf={"Dx geslacht": "Vrouw"})
-    samples = {sample.name: sample}
+    samples = {"test_sample": [SampleMock("test_sample", udf={"Dx geslacht": "Vrouw"})]}
```

Comment:
Does it matter that SampleMock("test_sample", udf={"Dx geslacht": "Vrouw"}) is a list in the new coding?